src/site/content/en/lighthouse-pwa/pwa-page-transitions/index.md

```diff
@@ -2,31 +2,43 @@
 layout: post
 title: Page transitions don't feel like they block on the network
 description: |
-  Learn about `pwa-page-transitions` audit.
-date: 2019-05-02
+  Learn how to make transitions between web pages feel responsive,
+  even on a slow network.
 web_lighthouse:
   - pwa-page-transitions
+date: 2019-05-04
+updated: 2019-09-19
 ---
 
-Transitions should feel snappy as you tap around, even on a slow network,
-a key to perceived performance.
+Quick page transitions are key to how users perceive the performance of your
+[Progressive Web App (PWA)](/discover-installable).
+Transitions should feel snappy, even on a slow network.
 
 ## Recommendations
```

Comment:
I created #1603 to remind ourselves to mention portals when they're more mature.